README.md

```diff
@@ -975,3 +977,10 @@ The `source_map_options` (optional) can contain the following properties:
   [codegen]: http://lisperator.net/uglifyjs/codegen
   [compressor]: http://lisperator.net/uglifyjs/compress
   [parser]: http://lisperator.net/uglifyjs/parser
+
+#### Support for `const`
+
+`const` in `uglify-js@2.x` has function scope and as such behaves much like `var`,
```

Comment:
The trailing comma above - could you please change it to a dash?

`const` in `uglify-js@2.x` has function scope and as such behaves much like `var` - unlike `const` in ES2015 (ES6) which has block scope.